back/app/controllers/web_api/v1/activities_controller.rb

```diff
@@ -1,37 +1,31 @@
 # frozen_string_literal: true
 
 class WebApi::V1::ActivitiesController < ApplicationController
-  before_action :set_post_type_and_id
-  skip_after_action :verify_policy_scoped
   skip_before_action :authenticate_user, only: %i[index]
 
   def index
-    activities = ActivitiesFinder.new(
```

Comment:
Think it's good that we've got rid of the `ActivitiesFinder` here. Seems an abstraction that was not needed given the simplicity of the queries here